notebooks/featurestore/hsfs/training/training-data-coalesced.ipynb

```diff
@@ -67,7 +67,7 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "Python",
    "language": "python",
    "name": "python3"
```

Comment:
I think this should also be a PySpark notebook.

Reply:
Fixed